hypothesis/h
# Add a separate docker image which only runs the Websocket

This also adds some fairly ugly commands to get this to run with:

   `make docker-ws` and `make run-docker-ws`

It's not really sensible to run this in anyway at the moment. Also currently this will build from the last uploaded version of the
`hypothesis/hypothesis` image rather than the local one.

### Testing notes

 * Run `make docker-ws` - This should build (based on an online version not the local one unfortunately)
 * Run `make run-docker-ws` - The image should start and keep running
 * Visit: http://localhost:5000/ws
 * You should see:
 
 ```json
 {
"ok": false,
"error": "bad_handshake",
"reason": "Handshake failed. Are you a WebSocket client?"
}
```
